### PR TITLE
Tg locations

### PR DESCRIPTION
**Pull request by MalarvizhiK**

### CODE REVIEW

File: website/ibm.erb [hunk 1]
```diff
@@ -406,6 +406,11 @@
               <a href="/docs/providers/ibm/d/tg_gateway.html">tg_gateway</a>
             </li>
           </ul>
+ 	  <ul class="nav nav-visible">
+            <li<%= sidebar_current("docs-ibm-datasource-tg-locations") %>>
+              <a href="/docs/providers/ibm/d/tg_locations.html">tg_locations</a>
```
Comment (hkantare) [inline website/ibm.erb:411]: create only 
```
<li<%= sidebar_current("docs-ibm-datasource-tg") %>>
          <a href="#">Transit Gateway Data Sources</a>
          <ul class="nav nav-visible">
            <li<%= sidebar_current("docs-ibm-datasource-tg-gateways") %>>
              <a href="/docs/providers/ibm/d/tg_gateways.html">tg_gateways</a>
            </li>
            <li<%= sidebar_current("docs-ibm-datasource-tg-gateway") %>>
              <a href="/docs/providers/ibm/d/tg_gateway.html">tg_gateway</a>
            </li>
            <li<%= sidebar_current("docs-ibm-datasource-tg-locations") %>>
              <a href="/docs/providers/ibm/d/tg_locations.html">tg_locations</a>
            </li>
          </ul>
        </li>
```

Comment (hkantare) [inline website/ibm.erb:411]: create only one ul section
```
<li<%= sidebar_current("docs-ibm-datasource-tg") %>>
          <a href="#">Transit Gateway Data Sources</a>
          <ul class="nav nav-visible">
            <li<%= sidebar_current("docs-ibm-datasource-tg-gateways") %>>
              <a href="/docs/providers/ibm/d/tg_gateways.html">tg_gateways</a>
            </li>
            <li<%= sidebar_current("docs-ibm-datasource-tg-gateway") %>>
              <a href="/docs/providers/ibm/d/tg_gateway.html">tg_gateway</a>
            </li>
            <li<%= sidebar_current("docs-ibm-datasource-tg-locations") %>>
              <a href="/docs/providers/ibm/d/tg_locations.html">tg_locations</a>
            </li>
          </ul>
        </li>
```

Comment (MalarvizhiK) [reply website/ibm.erb:411]: Corrected the changes.

Comment (MalarvizhiK) [reply website/ibm.erb:411]: corrected.

**Pull request merged**